lib/alchemy/upgrader/tasks/three_point_two_task.rb

```diff
@@ -0,0 +1,31 @@
+require 'thor'
+
+module Alchemy::Upgrader::Tasks
+  class ThreePointTwoTask < Thor
```

Comment:
Minor naming nitpick: Can we name it `Alchemy::Upgrader::Tasks::ThreePointTwo` rather than `...::ThreePointTwoTask` ?